content/en/apps/guides/hosting/4.x/self-hosting/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Self Hosting in CHT 4.x"
+linkTitle: "Self Hosting"
+weight: 10
+description: >
+  Details for hosting the CHT on self run infrastructure
+---
+
+# Recommendations and considerations
+
+## Multi vs Single node couchdb requirements
+
+For smaller deployments a [single node CouchDB][single-couch] instance can be used, for larger deployments a [multi-node CouchDB][multi-couch] cluster is generally recommended
+
+| Consideration                                   | [Single node CouchDB][single-couch] | [Multi-node clustered CouchDB][multi-couch] |
+| ----------------------------------------------- | ----------------------------------- | ------------------------------------------- |
+| Less than 4000 users                            | {{< icon/yes >}}                    | {{< icon/yes >}}                            |
+| More than 4000 users                            | {{< icon/no >}}                     | {{< icon/yes >}}                            |
+| Less than 10 000 documents per day              | {{< icon/yes >}}                    | {{< icon/yes >}}                            |
+| More than 10 000 documents per day              | {{< icon/no >}}                     | {{< icon/yes >}}                            |
```

Comment:
Nitpick: for consistency the "4000" number should have a thousand separator too. I also prefer commas to spaces because they don't line wrap, but it's also different in different locales so I'm not too worried about it.

```suggestion
| Less than 4,000 users                            | {{< icon/yes >}}                    | {{< icon/yes >}}                            |
| More than 4,000 users                            | {{< icon/no >}}                     | {{< icon/yes >}}                            |
| Less than 10,000 documents per day              | {{< icon/yes >}}                    | {{< icon/yes >}}                            |
| More than 10,000 documents per day              | {{< icon/no >}}                     | {{< icon/yes >}}                            |
```

Reply:
While I agree that comma is generally easier to read, especially for english speaking countries, I want to insist on avoiding them in international documentation as some locales use comma as decimal separator. I went with spaces seem to be the most accepted international standard [ref](https://en.wikipedia.org/wiki/Decimal_separator#Digit_grouping) and there's no risk of confusing it with a decimal separator.

I updated the PR to use the unicode character `Thin Space` which is thinner than a regular space in most fonts and should not word wrap.

Other options would be formatting numbers based on the browser locale using client side javascript, and/or using css for spacing to assist in functional copy paste, but I think that's overkill for this.

Reply:
> While I agree that comma is generally easier to read, especially for english speaking countries, I want to insist on avoiding them in international documentation as some locales use comma as decimal separator. I went with spaces as they seem to be the most accepted international standard [[ref](https://en.wikipedia.org/wiki/Decimal_separator#Digit_grouping)] and there's no risk of confusing it with a decimal separator.

Good point, well made! Would you mind updating the [style guide](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/) with this so we can stay consistent?

> Other options would be formatting numbers based on the browser locale using client side javascript, and/or using css for spacing to assist in functional copy paste, but I think that's overkill for this.

It seems crazy to me that there isn't a standard browser feature for this. Dates are even worse.

Reply:
> Would you mind updating the [style guide](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/) with this so we can stay consistent?

Sure, I'll open a new PR for this

@mrjones-plip this PR is blocked due to "changes requested", please give it a look when you have a moment